Eplant/views/ChromosomeViewer/index.tsx

```diff
@@ -82,6 +88,45 @@ const ChromosomeViewer: View<
   >) {
     const spaceRef = React.useRef<Space | null>(null)
     const [messageOpen, setMessageOpen] = useState(true)
+    const [annotationArray, setAnnotationArray] = useState<
+      GeneAnnotationItem[]
+    >([])
+
+    // Global State
+    const [geneticElements] = useGeneticElements()
+
+    //converts geneticElements into array of gene annotations
+    useEffect(() => {
+      const poplar = false
+      const species = poplar ? 'Populus_trichocarpa' : 'Arabidopsis_thaliana'
+      let newAnnotationArray: GeneAnnotationItem[] = []
+      geneticElements.map((gene) => {
```

Comment:
nit: you should use `forEach` in this instance, `map` is typically used for mapping an iterable to an array (e.g. const newArray = oldArray.map((e) => return something))